source/events/events.rst

```diff
@@ -115,10 +115,10 @@ An observatory Earth location should be given as well (see :ref:`coords-location
 * ``DEADC`` type: float
     * Dead time correction, defined by ``LIVETIME/ONTIME``.
       Is comprised in [0,1]. Defined to be 0 if ``ONTIME=0``.
-* ``RA_PNT`` type: float, unit: deg
-    * Pointing Right Ascension (see :ref:`coords-radec`).
-* ``DEC_PNT`` type: float, unit: deg
-    * Pointing declination (see :ref:`coords-radec`).
+* ``ALT_PNT`` type: float, unit: deg
```

Comment:
Making `ALT_PNT` / `AZ_PNT` required is not really a good idea, since IACTs track RA/Dec positions mostly.

Mabye we should require at least one of  `ALT_PNT`, `AZ_PNT` OR `RA_PNT`, `DEC_PNT` and specify how the telescope was tracking in a new required header keyword.

Reply:
That also sounds reasonable, something like `'POINTING_SYST'`, with the options `equatorial` or `horizontal`? Or maybe something more inspired, but along those lines

Reply:
`TRACKING = {ICRS/HORIZONTAL}` ?